crates/usvg/src/main.rs

```diff
@@ -426,7 +428,9 @@ fn process(args: Args) -> Result<(), String> {
     }?;
 
     let mut tree = usvg_tree::Tree::from_data(&input_svg, &re_opt).map_err(|e| format!("{}", e))?;
-    tree.convert_text(&fontdb);
+    if !args.preserve_text {
+        tree.convert_text(&fontdb);
```

Comment:
No need to load fontdb in this case either.

Reply:
`--list-font` shouldn't be used with input svg to begin with. A known bug. 
Let's leave it for later. The current implementation is fine.